### PR TITLE
Replace GH issue templates with forms

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

Forms are easier to be filled than writing plain text.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1200211978612931/1202958255787901/f) _(internal)_

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report-block.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        Please try to fill out the fields below. If you don't know what to put in the fields please left them blank.
+
+  - type: input
+    id: block
+    attributes:
+      label: Block impacted
+      description: The name of the block impacted.
+    validations:
+      required: true
```
Comment (TimDiekmann) [inline .github/ISSUE_TEMPLATE/bug-report-block.yml:21]: Unique for blocks. Should this be required?

File: .github/ISSUE_TEMPLATE/bug-report-hash-engine.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      required: false
+
+  - type: input
+    id: link
+    attributes:
+      label: Link to HASH Core
+      description: If your simulation is hosted at https://core.hash.ai/ and is public, please provide a link.
+      placeholder: e.g. https://core.hash.ai/@example/simulation
+    validations:
+      required: false
```
Comment (TimDiekmann) [inline .github/ISSUE_TEMPLATE/bug-report-hash-engine.yml:43]: Unique to HASH engine

File: .github/ISSUE_TEMPLATE/bug-report-hash-engine.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  - type: input
+    id: python-version
+    attributes:
+      label: Version
+      description: What Python version do you use?
+      placeholder: e.g. 3.10
+    validations:
+      required: false
```
Comment (TimDiekmann) [inline .github/ISSUE_TEMPLATE/bug-report-hash-engine.yml:87]: Added this section

**Review by TimDiekmann — COMMENTED** [review]
I highlighted the sections, which are different from other sections. Most sections are similar to the plain bug report but have added labels.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```
Comment (nonparibus) [inline .github/ISSUE_TEMPLATE/config.yml:1]:
```suggestion
blank_issues_enabled: true
```

**Review by nonparibus — APPROVED** [review]
I think we should leave `blank_issues_enabled` set to `true` for now, while the project structure is still evolving. Given this repo's catch-all status for our public projects it may even make sense to leave this setting intact indefinitely.

Otherwise this looks great. Thanks!

**Review by nonparibus — DISMISSED** [review]

**Comment by TimDiekmann** [comment]
I removed the title, as it was only `"[Bug]" `, but we already add a label to each to indicate that this is a bug.

**Review by nonparibus — APPROVED** [review]

**Pull request merged**